adafruit_is31fl3731/__init__.py

```diff
@@ -190,7 +190,7 @@ def autoplay(self, delay=0, loops=0, frames=0):
         self._register(_CONFIG_BANK, _AUTOPLAY2_REGISTER, delay % 64)
         self._mode(_AUTOPLAY_MODE | self._frame)
 
-    def fade(self, fade_in=None, fade_out=None, pause=0):
+    def fade(self, fade_in=None, fade_out=None, pause=26):
```

Comment:
Why change the pause default?

Reply:
https://github.com/adafruit/Adafruit_CircuitPython_IS31FL3731/blob/fa7437c8eafc48c60c4946bd8ba55a6dba56d68f/adafruit_is31fl3731/__init__.py#L212

```python
pause = int(math.log(pause / 26, 2))
```

throws another exception - log of 0 is _infinity_ - `26` is the "lowest default value" that doesn't automatically throw an error. I figured if a _user_ of the library sets a lower value and it barfs, that's on them: default values shouldn't cause errors, otherwise they're pretty useless :smiley:

Reply:
Wouldn't passing in a zero still result in the same problem? Perhaps only running pause through the math.log function if it is non-zero might work better? It seems that fade_in and fade_out would also probably suffer from the same issue if set to 0, which the docstring says is a valid value.

Reply:
Point taken! Modifications will be inbound in a bit...